airbyte-integrations/connectors/source-salesforce/source_salesforce/streams.py

```diff
@@ -252,18 +255,56 @@ def filter_null_bytes(self, s: str):
         """
         https://github.com/airbytehq/airbyte/issues/8300
         """
-        res = s.replace("\x00", "")
+        replace_list = ['"', "\x00", "\0", "\00"]
+        for rep in replace_list:
+            res = s.replace(rep, "")
```

Comment:
this will overwrite the replace from the previous elements.
It might be worth adding a test case for a string with all 4 null bytes and make sure they are all replaced

Reply:
sorry I did not looked closely first time to this place
```
>>> "\x00" == "\0" == "\00"
True
```

?